website/docs/cli/state/index.html.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: "docs"
+page_title: "Manipulating State - Terraform CLI"
+---
+
+# Manipulating Terraform State
+
+Terraform relies on [state data](/docs/state/index.html) to remember which
+real-world object corresponds to which managed resource in the configuration;
+this allows it to modify an existing object when changes are made to the
+corresponding resource declaration. For infrastructure managed by multiple
+people, the state data must be made available to all collaborators using a
+[backend](/docs/configuration/backend.html) that stores state remotely.
+
+Under normal conditions, Terraform updates state automatically during plans and
+applies. However, it's sometimes necessary to make deliberate adjustments to
+Terraform's state data, usually to compensate for changes to the configuration
+or the real managed infrastructure.
+
+When such situations arise, you can use one of Terraform CLI's commands for
+inspecting and modifying state data.
+
+Most of these commands are designed to help with a specific type of situation.
+
+~> **Important:** Modifying state data outside a normal plan or apply is
+inherently dangerous, because it can cause Terraform to lose track of managed
+resources. Depending on your provisioning practices and infrastructure usage,
+losing track of resources might cost you a lot of money, annoy all of your
+colleagues, or even compromise the security of your operations. Make sure to
+keep backups of your state data when modifying state out-of-band.
+
+## Checking Things Out
```

Comment:
Check out seems like an overloaded term. Can we say something more specific?

Reply:
"Inspecting State?"